### PR TITLE
Twitter Polygon

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <andrew.hulbert@ccri.com>

### CODE REVIEW

File: geomesa-tools/conf/sfts/twitter/reference.conf [hunk 1]
```diff
@@ -62,30 +151,44 @@ geomesa  {
...
+        { name = c3_lon, json-type = "string", path = "$.place.bounding_box.coordinates[0][3][0]", transform = "$0::double" }
+        { name = c3_lat, json-type = "string", path = "$.place.bounding_box.coordinates[0][3][1]", transform = "$0::double" }
+
+        { name = c0, transform = "concat( $c0_lon, concat(' ', $c0_lat))" }
+        { name = c1, transform = "concat( $c1_lon, concat(' ', $c1_lat))" }
+        { name = c2, transform = "concat( $c2_lon, concat(' ', $c2_lat))" }
+        { name = c3, transform = "concat( $c3_lon, concat(' ', $c3_lat))" }
+        { name = c4, transform = "concat( $c0_lon, concat(' ', $c0_lat))" }
+
+        { name = p0, transform = "concat( 'polygon((', $c0)"      }
```
Comment (elahrvivaz) [inline geomesa-tools/conf/sfts/twitter/reference.conf]: we should make concat accept more than 2 values...

**Comment by elahrvivaz** [comment]
looks good. @jahhulbert-ccri ok if i merge?

**Comment by jahhulbert-ccri** [comment]
@elahrvivaz ready to go i'll add a ticket for concat

**Comment by elahrvivaz** [comment]
550535ee2c5671ccc3245cf0caadc84a824ec2d8

**Pull request closed without merging**